modules/sublimeBidAdapter.js

```diff
@@ -66,7 +66,7 @@ export function sendEvent(eventName, sspName) {
     e: eventName,
     src: 'pa',
     puid: state.transactionId || state.notifyId,
-    trId: state.transactionId || state.notifyId,
+    notid: state.notifyId,
```

Comment:
```suggestion
    notid: state.notifyId || '',
```
car `state.notifyId` peut être vide suivant les intégrations

Reply:
Est-on certain que `trId` n’est plus nécessaire ?